avm/res/network/private-dns-zone/a/modules/nested_roleAssignments.bicep

```diff
@@ -0,0 +1,97 @@
+@sys.description('Required. The IDs of the principals to assign the role to.')
+param principalIds array
```

Comment:
This and all other `nested_roleAssignment.bicep` files are redundant now, right?

Reply:
removed - apologies on that one